dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -157,3 +171,164 @@ export const RenderPagination = (props) => {
     />
   );
 };
+
+export const EditRow = (props) => {
+  return (
+    <div className="pf-c-inline-edit__action pf-m-enable-editable">
+      {!props.item.isEdit ? (
+        <Button
+          variant="plain"
+          onClick={() => props.toggleEdit(props.item.resource_id, true)}
+          icon={<PencilAltIcon />}
+        />
+      ) : (
+        <div>
+          <Button
+            isDisabled={
+              !props.item.isDirty ||
+              !!!props.item.name ||
```

Comment:
What's `!!!`? Or is this just `!(!!props.item.name)` (boolean NOT of "convert value to boolean"); and, if so, why is it valuable to do the explicit conversion? (And GitHub's syntax coloring seems to suggest it's actually being grouped the other way, so this might be interpreted as `!!(!props.item.name)`.

Unless the tripled `!` actually means something entirely different that I wasn't able to find in a search, I'm not sure the explicit conversion to boolean is meaningful here. ❓❔

Reply:
Explicit conversion to boolean is not needed here